static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -160,33 +134,31 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
     return response.data;
   }, [api, orgSlug, projectSlug, replayId]);
 
-  const fetchSegmentList = useCallback(async () => {
-    const response = await api.requestPromise(
-      `/projects/${orgSlug}/${projectSlug}/replays/${replayId}/recording-segments/`
-    );
-    return response.data as ReplaySegment[];
-  }, [api, orgSlug, projectSlug, replayId]);
+  const fetchAllRRwebEvents = useCallback(async () => {
+    let next: ParsedHeader = {
+      href: `/projects/${orgSlug}/${projectSlug}/replays/${replayId}/recording-segments/?download&per_page=5`,
```

Comment:
i think we can crank per_page a lil higher to start. let's try 20 maybe?

Reply:
I took out the default of 5. that was a good number for test. 

Now there's no hard-coded per_page number on the client, so it'll pickup whatever the server's default is.